packages/sdk/src/plt/v1/Token.ts

```diff
@@ -222,8 +223,11 @@ export async function validateTransfer(
     const accounts = await Promise.all([senderPromise, ...receiverPromises]);
     accounts.forEach((r) => {
         const accToken = r.accountTokens.find((t) => t.id.value === token.info.id.value)?.state;
-        if (accToken?.memberDenyList || accToken?.memberAllowList === false) {
-            throw new NotAllowedError(r.accountAddress);
+        if (accToken?.moduleState !== undefined) {
+            const moduleState = Cbor.decode(accToken.moduleState) as TokenModuleAccountState;
+            if (moduleState.memberDenyList || moduleState.memberAllowList === false) {
+                throw new NotAllowedError(r.accountAddress);
+            }
```

Comment:
This is a nitpick,  but would be nice to reduce nesting:
```suggestion
        if (accToken?.moduleState === undefined) {
            return;
        }
        
        const moduleState = Cbor.decode(accToken.moduleState) as TokenModuleAccountState;
        if (moduleState.memberDenyList || moduleState.memberAllowList === false) {
            throw new NotAllowedError(r.accountAddress);
        }
```